src/rules/string-quotes/__tests__/index.js

```diff
@@ -126,3 +126,29 @@ testRule(rule, {
     column: 12,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: ["double"],
+  skipBasicChecks: true,
+  syntax: "less",
+
+  accept: [{
+    code: "a {\n  // 'horse'\n}",
+    description: "ignores single-line LESS comment",
```

Comment:
So is Less capitalized or ALL CAPS? I gather from the website it's just capitalized.